specification/eventgrid/data-plane/Microsoft.ResourceNotifications/stable/2018-01-01/common.json

```diff
@@ -0,0 +1,105 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2018-01-01",
+    "title": "Schema of the common properties for Azure Resource Notifications (ARN) System topics",
+    "description": "Describes the schema of the the common properties across all topic types published by ARN to Azure Event Grid. This corresponds to the Data property of an EventGridEvent."
+  },
+  "paths": {},
+  "definitions": {
+    "ResourceNotificationsResourceWithPayloadDetails": {
```

Comment:
Should this be removed?

Reply:
No since this is being used in ResourceNotificationsResourceUpdatedDetails which is turn being referenced in HealthResources.json

Reply:
This should be a base event that references ResourceNotificationsOperationalDetails and ResourceNotificationsResourceUpdatedDetails as properties.